lib/ansiblelint/__main__.py

```diff
@@ -77,9 +99,40 @@ def main():
                       help="Try force colored output (relying on ansible's code)")
     parser.add_option('--exclude', dest='exclude_paths', action='append',
                       help='path to directories or files to skip. This option'
-                           ' is repeatable.')
+                           ' is repeatable.',
+                      default=[])
+    parser.add_option('-F', help='Specify configuration file to use.  Defaults to ".ansible-lint"')
     options, args = parser.parse_args(sys.argv[1:])
 
+    config = load_config(options.F)
+
+    if config:
+        if 'quiet' in config:
+            options.quiet = options.quiet or config['quiet']
+
+        if 'parseable' in config:
+            options.parseable = options.parseable or config['parseable']
+
+        if 'use_default_rules' in config:
+            options.use_default_rules = \
+                options.use_default_rules or config['use_default_rules']
+
+        if 'verbosity' in config:
+            options.verbosity = options.verbosity + config['verbosity']
+
+        if 'exclude_paths' in config:
+            options.exclude_paths = \
```

Comment:
`max-line-length` is set to 100 in setup.cfg so we don't have to do things like this :)